mozregression/telemetry.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import sys
+from multiprocessing import Process
+
+from mozlog import get_proxy_logger
+
+
+LOG = get_proxy_logger('telemetry')
+
+
+def _send_telemetry_ping(variant, appname):
+    from pathlib import Path
+    from glean import Glean, Configuration
+    from glean import (load_metrics,
+                       load_pings)
+    from mozregression import __version__
+
+    mozregression_path = Path.home() / '.mozilla' / 'mozregression'
+    Glean.initialize(
+        application_id="mozregression",
+        application_version=__version__,
+        upload_enabled=True,
+        configuration=Configuration(
+            ping_tag="mozregression-test-tag"
+        ),
+        data_dir=mozregression_path / "data"
+    )
+
+    # if we got this far, we can assume telemetry is enabled
+    Glean.set_upload_enabled(True)
```

Comment:
This is redundant to the `upload_enabled` argument to `Glean.initialized`.  However, this should be called from the callback to the telemetry checkbox in the GUI so that Glean can clear metrics, reset the client id and send a deletion ping request if necessary.

Reply:
Sending the deletion ping doesn't seem to be working (at least with test tags and the debug ping viewer), but I've deleted this line.